core/src/main/java/com/gentics/mesh/core/endpoint/project/LanguageEndpoint.java

```diff
@@ -39,22 +43,61 @@ public String getDescription() {
 	public void registerEndPoints() {
 		secureAll();
 
-		// TODO Add method that allows assigning languages from and to the project
-		Route createRoute = route("/:projectUuid/languages").method(POST).produces(APPLICATION_JSON);
-		createRoute.handler(rc -> {
-			throw new NotImplementedException("not implemented");
-		});
-
-		Route deleteRoute = route("/:projectUuid/languages").method(DELETE).produces(APPLICATION_JSON);
-		deleteRoute.handler(rc -> {
-			// Unassign languages should cause a batch process that removes the FieldContainers for the given language.
-			throw new NotImplementedException("not implemented");
-		});
-
-		Route getRoute = route("/:projectUuid/languages").method(GET).produces(APPLICATION_JSON);
-		getRoute.handler(rc -> {
-			throw new NotImplementedException("not implemented");
-		});
-	}
+		InternalEndpointRoute createRoute = createRoute();
```

Comment:
Does it make sense to add the endpoints for creating/deleting, which will always return errors?
If the answer is yes, shouldn't we add endpoints for updating as well (also returning errors?)
imho, we should remove the unsupported endpoints completely.